week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,11 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(clean); // Complete this statement
+
+function clean(elem) {
```

Comment:
This fails to filters nulls and arrays with length that is not 2.
```suggestion
return Array.isArray(item) && (item.length == 2);
```